docs/api/vega-wallet/reference/browser-extension/get-started.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Build with the browser extension 
+hide_title: false
+vega_network: TESTNET
+sidebar_position: 1
+---
+import Topic from '/docs/topics/_topic-wallet.mdx'
+
+<Topic />
+
+## Vega Browser Wallet
+
+dApps and websites can communicate with the Vega browser wallet over 
+JSON-RPC 2.0, however the browser extension also provides
+a high-level Javascript API, injected into web pages.
+
+### Detecting the extension
+
+The Vega browser wallet injects methods on the global `vega` object,
+hence the presence of the extension can be detected by checking that
+this object is defined:
+
+```js
+if (window.vega != null) {
+    // Vega is installed
+} else {
+    // Prompt the user to install the extension
+}
+```
+
+### Connecting to the extension
+
+To start communicating with the extension, a dApp must first request
+a connection:
+
+```js
+try {
+    await vega.connectWallet()
+} catch (err) {
+    // The user potentially rejected the connection    
+}
+```
+
+The first time a dApp tries to connect to a wallet, the user will be
+prompted to approve the connection and select which keys they will
+allow the dApp to see. Only appoved keys will be available to the dApp.
+
+Subsequent connect attempts will resolve automatically, unless the user
+has removed the connection in the meantime.
+
+Once a dApp is finished talking to the wallet it can call 
```

Comment:
Something to consider - what about adding guidance or suggestions on when it would be a good time to automatically 'disconnect wallet'?

Reply:
Yeah, I don't actually know if you would ever do that automatically or just offer it as an option to the user. I at least can't think of a reason other than to be "nice" if the dApp does some kind of one-off transaction